Frontend/src/App.jsx

```diff
@@ -1,10 +1,20 @@
 import "./App.css";
-import Signup from "./components/Signup";
+
+import Footer from "./components/Footer";
+import i18n from "./components/I18n";
+import Header from "./components/Header";
+import Body from "./components/Body";
 
 function App() {
   return (
     <>
-      <Signup />
+      <div className="min-h-screen bg-white">
+        <Header />
+        <div style={{ marginTop: "-50%" }}>
```

Comment:
Why the fuck would you use inline styles here ? Forgot about tailwind? WHAT THE FUCK